code/backend/batch/utilities/helpers/llm_helper.py

```diff
@@ -154,3 +156,34 @@ def get_sk_service_settings(self, service: AzureChatCompletion):
                 max_tokens=self.llm_max_tokens,
             ),
         )
+
+    def get_ml_client(self):
+        if not hasattr(self, "_ml_client"):
+            self._ml_client = MLClient(
+                DefaultAzureCredential(),
+                self.env_helper.AZURE_SUBSCRIPTION_ID,
+                self.env_helper.AZURE_RESOURCE_GROUP,
+                self.env_helper.AZURE_ML_WORKSPACE_NAME,
+            )
+        return self._ml_client
+
+    def get_endpoint_name(self):
+        return self.env_helper.PROMPT_FLOW_ENDPOINT_NAME
+
+    def get_deployment_name(self):
+        return self.env_helper.PROMPT_FLOW_DEPLOYMENT_NAME
+
+    def transform_chat_history_for_pf(self, chat_history):
```

Comment:
Just wondering if this should be a method within the `PromptFlowOrchestrator`? I think it'll only ever be used there.